python-iml-manager.spec

```diff
@@ -95,48 +94,14 @@ Requires:       rust-iml-cli
 Requires:       influxdb
 Requires:       grafana
 
-Conflicts:      chroma-agent
+Conflicts:       chroma-agent
```

Comment:
Do we need the extra space?

Reply:
Just making it line up with the other lines.

Reply:
ah ok it looked like it was spaced over in the pr.